docs/_future/manifest_functions.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Manifest functions
+order: 3
+slug: manifest-functions
+lang: en
+layout: future
+---
+
+<div class="section-content">
+Your app can [listen to Functions](/bolt-python/future/concepts#functions) defined and created by you. In order for this to work, Slack must know they exist. Define them in your [App Manifest](/bolt-python/concepts#manifest) also known as `manifest.json` in your project. The next time you `slack run` your app will inform Slack they exist.
```

Comment:
Hmm, listen to functions? Or invoke functions?

Reply:
I like "invoke" over "listen", I think I used "listen" because that is the pattern in the docs for the other events